cmd/get_secret.go

```diff
@@ -114,10 +118,20 @@ func GetSecretDescription(clientSet kubernetes.Interface, secretName string) (se
 		CoreV1().
 		Secrets(Namespace).
 		List(listOpts)
+
 	if err != nil {
 		fmt.Printf("Failed to list secrets in namespace %v: %v\n", Namespace, err.Error())
 		return nil, err
 	}
+
+	// Due to an issue with the ListOptions filters in
+	// kubernetes fake-client that is used for testing
+	// we cannot check for exactly one result
```

Comment:
Wow!

Reply:
One should probably not write bad code because of things like this but I see it very unlikely that the real kubernetes API would fail on this point.